src/sentry/tasks/signals.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+
+from sentry.plugins import plugins
+from sentry.tasks.base import instrumented_task
+from sentry.utils.safe import safe_execute
+
+
+@instrumented_task(name='sentry.tasks.signal')
```

Comment:
Is this a bit generically named? Are you expecting the implementation here to expand beyond just signaling plugins? Otherwise it seems like a more appropriate name would be `signal_plugins`. (maybe you were just trying to avoid the word plugin?)

Reply:
its generically named as it could potentially be used for non-plugins (granted, i dont know we'll ever want to given the move to pubsub)